FAQ.md

```diff
@@ -81,6 +81,18 @@ app is not possible.
 
 Cloud providers currently known to be supported:
 - Nextcloud
+- Google: Android Settings > Backup > Back up now; it's [on your Google Drive](https://drive.google.com/drive/u/0/backups)
+
+When migrating to a new phone, just install Aegis again, and the Cloud provider
```

Comment:
I just migrated to a new phone and when starting Aegis I had the option to import a file or follow the regular setup asking me to provide a password. Did I miss something here? Can I trigger the backup restore manual or should this happen automatically.

Reply:
@hofbi would you like to Resolve this code review comment?

Reply:
Yes let's resolve that. I added a suggestion below for my findings.